src/main/java/com/exactpro/th2/storageservices/model/KeyspaceResponse.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2020-2022 Exactpro (Exactpro Systems Limited)
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.exactpro.th2.storageservices.model;
+
+import com.datastax.oss.driver.api.core.CqlIdentifier;
+import com.datastax.oss.driver.api.core.metadata.schema.KeyspaceMetadata;
+import io.swagger.v3.oas.annotations.media.Schema;
+
+@Schema(name = "keyspaceResponse", title = "keyspaceResponse", description = "Keyspace response class")
+public class KeyspaceResponse {
+
+    private final CqlIdentifier name;
+
+    private final boolean durableWrites;
```

Comment:
Durable writes info might not be needed in response, I think it can be deleted

Reply:
removed